### DISCUSSION TITLE
How to rename multiple columns in Vaex Dataframe

### DISCUSSION THREAD

**Discussion by omonmaxi** [vaexio/vaex#1923] [Q&A]
I've tried using the rename method and found out it only works for a single column. I tried passing a dictionary just as in pandas but that didn't work. I would appreciate if you could point me in the right direction. Thanks

**Comment by JovanVeljanoski** [comment]
This is tracked:
 - https://github.com/vaexio/vaex/pull/1135
 - https://github.com/vaexio/vaex/pull/1795 (probably a better attempt).

**Reply by omonmaxi** [reply]
@JovanVeljanoski  Thanks for letting me know this. By the way, thanks you all for this GREAT package.
